src/pages/projectSettings/tabs/GithubCommitQueueTab/GithubCommitQueueTab.tsx

```diff
@@ -38,6 +45,19 @@ export const GithubCommitQueueTab: React.FC<TabProps> = ({
   const { getTab, updateForm } = useProjectSettingsContext();
   const { formData } = getTab(tab);
 
+  const { data } = useQuery<
+    GithubProjectConflictsQuery,
+    GithubProjectConflictsQueryVariables
+  >(GET_GITHUB_PROJECT_CONFLICTS, {
+    skip: projectType === ProjectType.Repo,
+    variables: { projectId: identifier },
+    onError: (e) => {
+      reportError(
+        new Error(`Error fetching GitHub Project Conflicts: ${e}`)
+      ).warning();
+    },
```

Comment:
graphql automatically reports errors from queries so this would double report is this necessary? Should we show an error to the user in this case?